app/scripts/directives/processTemplate.js

```diff
@@ -226,7 +228,10 @@
         };
         ctrl.template.labels = keyValueEditorUtils.mapEntries(keyValueEditorUtils.compactEntries(ctrl.labels));
 
-        DataService.create("processedtemplates", null, ctrl.template, context).then(
+        var tplGroupVersion = APIService.objectToResourceGroupVersion(ctrl.template);
```

Comment:
This reflects the latest small change we discussed.

@sg00dwin this branch updated, you might want to pull again if you are still using it (but I imagine it works for what you are doing either way).

Reply:
@spadgett I think this covers the small change from the last discussion we had.

Reply:
nit: `tpl` = template? I'd rather just call it `rgv` or `resourceGroupVersion`

Reply:
We should add a comment here explaining that the processedtemplates version must match the version of the template in the request since it's not obvioius why we're doing this.

Reply:
Good with a named function?

```JavaScript  
  //
  var rgv = getProcessTemplateVersionForTemplate(ctrl.template);

  DataService.create(rgv, null, ctrl.template, context).then(
  //  
```